### ISSUE TITLE
Use of deprecated numpy.testing.decorators

### ISSUE THREAD

**Issue by avaruuser** [Eelbrain/Eelbrain#38]
Hello,

I'm getting the following error if I am trying to import eelbrain on a newly installed system. It seems that it is trying to use a deprecated function of numpy:

```
Traceback (most recent call last):
  File "pythonscript.py", line 1, in <module>
    from eelbrain import *
  File "/home/user/miniconda3/envs/eelbrain/lib/python3.7/site-packages/eelbrain/__init__.py", line 16, in <module>
    from . import mne_fixes
  File "/home/user/miniconda3/envs/eelbrain/lib/python3.7/site-packages/eelbrain/mne_fixes/__init__.py", line 38, in <module>
    from ._label import write_labels_to_annot
  File "/home/user/miniconda3/envs/eelbrain/lib/python3.7/site-packages/eelbrain/mne_fixes/_label.py", line 11, in <module>
    import nibabel
  File "/home/user/.local/lib/python3.7/site-packages/nibabel/__init__.py", line 38, in <module>
    from . import analyze as ana
  File "/home/user/.local/lib/python3.7/site-packages/nibabel/analyze.py", line 87, in <module>
    from .volumeutils import (native_code, swapped_code, make_dt_codes,
  File "/home/user/.local/lib/python3.7/site-packages/nibabel/volumeutils.py", line 22, in <module>
    from .casting import (shared_range, type_info, OK_FLOATS)
  File "/home/user/.local/lib/python3.7/site-packages/nibabel/casting.py", line 11, in <module>
    from .testing import setup_test  # flake8: noqa F401
  File "/home/user/.local/lib/python3.7/site-packages/nibabel/testing/__init__.py", line 21, in <module>
    from numpy.testing.decorators import skipif
ModuleNotFoundError: No module named 'numpy.testing.decorators'
```

I'm using versions 
python 3.7.6
numpy 1.18.1
eelbrain 0.29.8

is there a fix to this?

thanks

**Comment by christianbrodbeck** [comment]
It looks like this is fixed in nibabel 3.0.1, can you try updating nibabel? I'll also update the nibabel version requirement in the eelbrain conda-forge recipe.

**Comment by christianbrodbeck** [comment]
Also just noticed that your version of Eelbrain is several versions back, is there another dependency in the environment that is holding it back?

**Comment by avaruuser** [comment]
> It looks like this is fixed in nibabel 3.0.1, can you try updating nibabel? I'll also update the nibabel version requirement in the eelbrain conda-forge recipe.

That indeed solved the issue, thanks a lot!
